### ISSUE TITLE
[question] how to make default security be warning not error

### ISSUE THREAD

**Issue by Freed-Wu** [iamcco/coc-diagnostic#126]
Such as

```json
      "lacheck": {
        "command": "lacheck",
        "args": ["%file"],
        "debounce": 100,
        "offsetLine": 0,
        "offsetColumn": 0,
        "sourceName": "lacheck",
        "formatLines": 1,
        "formatPattern": [
          "\"[^\"]+\", line (\\d+):\\s*(.*)(\\r|\\n)*$",
          {
            "line": 1,
            "message": 2
          }
        ],
      }
```

and

```shell
$ lacheck main.tikz
"main.tikz", line 10: possible unwanted space at "{"
```

`CocDiagnostics` will report `	[lacheck]		Error	possible unwanted space at "{"`

However, according to `lacheck --help`, all lacheck's messages are warning:

```shell
$ lacheck --help|rg warning
All messages are only warnings!
```

So the right `CocDiagnostics` should be `	[lacheck]		Warning	possible unwanted space at "{"`

How to change the default security? Thanks!

**Comment by iamcco** [comment]
try https://github.com/iamcco/coc-diagnostic/pull/56#issuecomment-711620863
